llama-index-integrations/readers/llama-index-readers-astra-db/tests/test_astra_db.py

```diff
@@ -0,0 +1,73 @@
+import os
+import pytest
+from typing import Iterable
+
+import astrapy
+from astrapy.db import AstraDB
+from llama_index.readers.astra_db import AstraDBReader
+from llama_index.core.schema import Document
+
+COLLECTION_NAME = "li_readers_test"
+
+print(f"astrapy detected: {astrapy.__version__}")
+
+
+# env variables
+ASTRA_DB_APPLICATION_TOKEN = os.getenv("ASTRA_DB_APPLICATION_TOKEN", "")
+ASTRA_DB_API_ENDPOINT = os.getenv("ASTRA_DB_API_ENDPOINT", "")
+
+
+@pytest.fixture(autouse=True, scope="module")
+def source_collection() -> Iterable[str]:
```

Comment:
Type is a collection

Reply:
Thank you - fix is landed along with a couple other things